src/main.cpp

```diff
@@ -2453,7 +2453,15 @@ void vcMain_RenderSceneWindow(vcState *pProgramState)
           allowedControls = (vcGizmoAllowedControls)(allowedControls & ((vcSceneItem*)ref.pItem->pUserData)->GetAllowedControls());
 
         //read direction axes again.
-        vcGIS_GetOrthonormalBasis(pProgramState->geozone, pProgramState->camera.position, &pProgramState->gizmo.direction[2], &pProgramState->gizmo.direction[1], &pProgramState->gizmo.direction[0]);
+        if (pProgramState->gizmo.operation == vcGO_Scale || pProgramState->gizmo.coordinateSystem == vcGCS_Local)
+        {
+          pProgramState->gizmo.direction[0] = udDouble3::create(1, 0, 0);
+          pProgramState->gizmo.direction[1] = udDouble3::create(0, 1, 0);
+          pProgramState->gizmo.direction[2] = udDouble3::create(0, 0, 1);
+        }
+        else
```

Comment:
The else should also have braces - the coding standard says something along the lines of: if the `if` has braces the `else` must have braces too.

Reply:
fixed.